roles/openshift_master_facts/tasks/main.yml

```diff
@@ -0,0 +1,145 @@
+---
+- name: Check for RPM generated config marker file .config_managed
+  stat:
+    path: /etc/origin/.config_managed
+  register: rpmgenerated_config
+
+- name: Remove RPM generated config files if present
+  file:
+    path: "/etc/origin/{{ item }}"
+    state: absent
+  when: rpmgenerated_config.stat.exists == true and deployment_type in ['openshift-enterprise', 'atomic-enterprise']
+  with_items:
+  - master
+  - node
+  - .config_managed
+
+- set_fact:
+    openshift_master_pod_eviction_timeout: "{{ lookup('oo_option', 'openshift_master_pod_eviction_timeout') | default(none, true) }}"
+  when: openshift_master_pod_eviction_timeout is not defined
```

Comment:
instead of using set_fact here, you can just set this using the default filter in the openshift_facts task below.